scenario/context/CometContext.ts

```diff
@@ -52,6 +52,18 @@ export class CometContext {
   actors: ActorMap;
   assets: AssetMap;
 
+  // COMP biggest whales
+  // XXX find a better way to do this without hardcoding whales
+  voters = [
+    '0xea6c3db2e7fca00ea9d7211a03e83f568fc13bf7',
+    '0x61258f12c459984f32b83c86a6cc10aa339396de',
+    '0x9aa835bc7b8ce13b9b0c9764a52fbf71ac62ccf1',
+    '0x683a4f9915d6216f73d6df50151725036bd26c02',
+    '0xa1b61405791170833070c0ea61ed28728a840241',
+    '0x88fb3d509fc49b515bfeb04e23f53ba339563981',
+    '0x8169522c2c57883e8ef80c498aab7820da539806'
+  ];
```

Comment:
We should figure out how to reconcile this list for `mainnet` with the sourced ones 🥈 will be working on

Reply:
I can do in separate pr once it's done